### PR TITLE
inequality module

### PR DESCRIPTION
**Pull request by fpiontek**
# Purpose of this PR
add the inequality module with its links to climate policy and damages

## Type of change
- [ ] New feature 
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by LaviniaBaumstark — COMMENTED** [review]
why is [modules/51_internalizeDamages/KWlikeItrCPreg/datainput.gms] an empty file?

File: main.gms [hunk 1]
```diff
@@ -696,6 +697,13 @@ parameter
 ;
   cm_fetaxscen        = 3;         !! def = 3
 *'
+parameter
+  cm_distrBeta              "elasticity of tax revenue redistribution"
+*** (0): equal per capita redistribution
```
[the remark this answers is not in the document]
Comment (fpiontek) [reply main.gms:702]: done

File: core/declarations.gms [hunk 2]
```diff
@@ -15,6 +15,9 @@ parameters
 
 ***----------------------------------------------------------------------------------------
 ***--------------------------------------------------MACRO module--------------------------
+***
+*pm_cons(ttot,all_regi)                              "consumption parameter (needed for some tests)"
```
[the remark this answers is not in the document]
Comment (fpiontek) [reply core/declarations.gms:19]: done

File: core/equations.gms [hunk 3]
```diff
@@ -211,7 +211,44 @@ qm_balFe(t,regi,entySe,entyFe,te)$se2fe(entySe,entyFe,te)..
   sum((sector2emiMkt(sector,emiMkt),entyFE2sector(entyFE,sector)),
     vm_demFEsector(t,regi,entySE,entyFE,sector,emiMkt)
   )
-; 
+;
+
+***---------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (fpiontek) [reply core/equations.gms:216]: done

**Review by LaviniaBaumstark — APPROVED** [review]

**Comment by orichters** [comment]
Congrats, Franziska! Regarding "why is [modules/51_internalizeDamages/KWlikeItrCPreg/datainput.gms] an empty file?": The diff is empty, only access rights were corrected. The text "empty file" is very counterintuitive.

**Pull request merged**